packages/web-app-user-management/src/components/Groups/GroupsList.vue

```diff
@@ -96,6 +104,13 @@ export default {
           name: 'members',
           title: this.$gettext('Members'),
           sortable: true
+        },
+        {
+          name: 'actions',
+          title: '',
```

Comment:
Any reason for the empty title? `this.$gettext('Actions')`

Reply:
Shown in scribbles, let’s check